### PR TITLE
Correct several problems in i8kmon

### PR DESCRIPTION
**Pull request by x-hgg-x**
Disable setting of fan speed before probing CPU temperature.
Add possibility to add more temperature configurations.
Force fan speed modification if the config state is inconsistent with fan speed.
Don't read fan state from BIOS.

### CODE REVIEW

**Comment by sabit** [comment]
Audio stuttering gone on Dell G7 15. Thanks! Hope this becomes part of master.

File: debian/i8kmon.service [hunk 1]
```diff
@@ -4,7 +4,11 @@ Documentation=man:i8kmon
 ConditionPathExists=/proc/i8k
 
 [Service]
+ExecStartPre=/usr/local/bin/dell-bios-fan-control/dell-bios-fan-control 0
```
[the remark this answers is not in the document]
Comment (vitorafsr) [reply debian/i8kmon.service]: As "dell-bios-fan-control" is not in the package now, this modification is not adequate, so it is not going to be merged by now.

**Review by vitorafsr — APPROVED** [review]

File: debian/i8kmon.service (hunk 1 above)
[the remark this answers is not in the document]
Comment (x-hgg-x) [reply debian/i8kmon.service]: We can add the same lines as in the redhat service : https://github.com/vitorafsr/i8kutils/blob/master/redhat/i8kmon.service#L6_L8

File: i8kmon [hunk 2]
```diff
@@ -24,10 +24,12 @@ array set config {
     verbose	0
     timeout	2
     t_high	80
+    num_configs	5
```
[the remark this answers is not in the document]
Comment (x-hgg-x) [reply i8kmon]: My laptop has 2 fans, one for the CPU and the other for the discrete GPU.

**Pull request merged**